### PR TITLE
improve project setup

### PR DESCRIPTION
**Pull request by jairhenrique**
Fix https://github.com/argaen/aiocache/issues/329

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,36 +14,42 @@
...
+    return install_requires
 
-if not strtobool(os.environ.get("AIOCACHE_REDIS", "yes")):
-    print("Installing without aioredis")
-    install_requires.remove(aioredis)
 
-if not strtobool(os.environ.get("AIOCACHE_MEMCACHED", "yes")):
-    print("Installing without aiomcache")
-    install_requires.remove(aiomcache)
+requirements = load_requirements('requirements.txt')
```
[the remark this answers is not in the document]
Comment (jairhenrique) [reply setup.py]: 👍

**Comment by argaen** [comment]
Thanks!

I'm keeping #329 open because I want to move dev dependencies into setup too.

**Comment by jairhenrique** [comment]
@argaen setuptools have a section to define development/tests dependencies? 🤔

**Comment by argaen** [comment]
It has for test dependencies but my intention is to use `extras_require` as Flask does: https://github.com/pallets/flask/blob/master/setup.py#L79.

**Pull request merged**